I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 182, in script call
    call intro from _call_intro
  File "game/intro.rpy", line 96, in script
    mauh "Great! Oh, and oops (character shake), I forgot to introduce myself silly me. I’m Mauh, your friendly neighborhood happy-go-lucky popular girl! And this is..."
NameError: name 'Mauh' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 182, in script call
    call intro from _call_intro
  File "game/intro.rpy", line 96, in script
    mauh "Great! Oh, and oops (character shake), I forgot to introduce myself silly me. I’m Mauh, your friendly neighborhood happy-go-lucky popular girl! And this is..."
  File "C:\Users\Russel\Downloads\renpy-8.3.3-sdk\renpy\ast.py", line 2573, in execute
    Say.execute(self)
  File "C:\Users\Russel\Downloads\renpy-8.3.3-sdk\renpy\ast.py", line 615, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\Users\Russel\Downloads\renpy-8.3.3-sdk\renpy\exports\sayexports.py", line 132, in say
    who(what, *args, **kwargs)
  File "C:\Users\Russel\Downloads\renpy-8.3.3-sdk\renpy\character.py", line 1438, in __call__
    who = self.prefix_suffix("who", self.who_prefix, who, self.who_suffix)
  File "C:\Users\Russel\Downloads\renpy-8.3.3-sdk\renpy\character.py", line 1354, in prefix_suffix
    return (sub(prefix) + sub(body) + sub(suffix))
  File "C:\Users\Russel\Downloads\renpy-8.3.3-sdk\renpy\character.py", line 1334, in sub
    return renpy.substitutions.substitute(s, scope=scope, force=force, translate=translate)[0]
  File "C:\Users\Russel\Downloads\renpy-8.3.3-sdk\renpy\substitutions.py", line 373, in substitute
    s = interpolate(s, variables) # type: ignore
  File "C:\Users\Russel\Downloads\renpy-8.3.3-sdk\renpy\substitutions.py", line 86, in interpolate
    raise e
  File "C:\Users\Russel\Downloads\renpy-8.3.3-sdk\renpy\substitutions.py", line 78, in interpolate
    value = renpy.python.py_eval(code, {}, scope)
  File "C:\Users\Russel\Downloads\renpy-8.3.3-sdk\renpy\python.py", line 1209, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\Russel\Downloads\renpy-8.3.3-sdk\renpy\python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "<none>", line 1, in <module>
NameError: name 'Mauh' is not defined

Windows-10-10.0.22631 AMD64
Ren'Py 8.3.3.24111502
Wumauhen 1.0
Fri Nov 22 17:12:23 2024
